climada/__init__.py

```diff
@@ -18,7 +18,89 @@
 
 climada init
 """
+from shutil import copyfile
+from pathlib import Path
 
-from .util.config import CONFIG, setup_conf_user, setup_logging
-setup_conf_user()
-setup_logging(CONFIG['global']['log_level'])
+from .util.config import CONFIG, setup_logging
+from .util.constants import *
```

Comment:
Is this * import to be temporary or will constants remain in the util? I thought constant should be defined in the config file right?